eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/ChannelTlv.scala

```diff
@@ -107,10 +108,17 @@ object FundingSignedTlv {
   val fundingSignedTlvCodec: Codec[TlvStream[FundingSignedTlv]] = tlvStream(discriminated[FundingSignedTlv].by(varint))
 }
 
-sealed trait FundingLockedTlv extends Tlv
+sealed trait ChannelReadyTlv extends Tlv
 
-object FundingLockedTlv {
-  val fundingLockedTlvCodec: Codec[TlvStream[FundingLockedTlv]] = tlvStream(discriminated[FundingLockedTlv].by(varint))
+object ChannelReadyTlv {
+
+  case class ShortChannelIdTlv(alias: ShortChannelId) extends ChannelReadyTlv
```

Comment:
Shouldn't this be a `LocalAlias` instead of a `ShortChannelId`? Actually this raises a more general question: why did you define a `LocalAlias` instead of a more general `ShortChannelIdAlias` used for both local and remote? What they send us in here should be an alias, right?